posthog/settings/__init__.py

```diff
@@ -117,6 +117,3 @@
 
 # Lastly, cloud settings override and modify all
 from posthog.settings.cloud import *
-
-# TODO: Temporary
-EMAIL_REPORTS_ENABLED: bool = get_from_env("EMAIL_REPORTS_ENABLED", False, type_cast=str_to_bool)
```

Comment:
Pet peeve: avoid this next time - __init__ file should be as clean as possible, avoid littering random settings at the bottom (especially if the line above says "Lastly).